packages/components/package.json

```diff
@@ -133,5 +133,9 @@
 		"doc",
 		"custom-elements.json",
 		"vscode-custom-data.json"
-	]
+	],
+	"lint-staged": {
+		"*.ts|tsx|js|jsx": ["prettier --list-different src", "pnpm lint"],
```

Comment:
Wenn ich es teste, scheint es mit mit `pnpm lint` nicht korrekt zu funktionieren.
Mein Setup:

- Ich füge Linting-Fehler in zwei .ts-Dateien im Components-Package ein.
- Ich stage nur eine der Dateien.
- Alle Dateien im Package werden gelintet, beide Fehler ausgegeben.

Meine Vorstellung wäre gewesen, dass nur die "staged" Datei gelintet wird.

![screenshot 2024-08-26-12 11 44@2x](https://github.com/user-attachments/assets/66225a08-0081-45e1-991f-1470a75c3b40)